tests/aws/services/es/test_es.py

```diff
@@ -86,7 +86,7 @@ def test_get_compatible_versions(self, aws_client):
 
         versions = response["CompatibleElasticsearchVersions"]
 
-        assert len(versions) == 24
+        assert len(versions) == 26
```

Comment:
This will become 25 (after the removal of 2.10).
Please also remove the comment in line 124 stating that 2.9 is the current latest version.